command/resource/client/grpc-flags.go

```diff
@@ -5,54 +5,60 @@ package client
 
 import (
 	"flag"
+	"strings"
 )
 
 type GRPCFlags struct {
-	address StringValue
+	address  StringValue
+	grpcTLS  BoolValue
+	certFile StringValue
+	keyFile  StringValue
+	caFile   StringValue
+	caPath   StringValue
 }
 
-// mergeFlagsIntoGRPCConfig merges flag values into grpc config
+// MergeFlagsIntoGRPCConfig merges flag values into grpc config
 // caller has to parse the CLI args before loading them into flag values
-func (f *GRPCFlags) mergeFlagsIntoGRPCConfig(c *GRPCConfig) {
+// flag values take precedence over the environment values
+func (f *GRPCFlags) MergeFlagsIntoGRPCConfig(c *GRPCConfig) {
+	if strings.HasPrefix(strings.ToLower(f.address.String()), "https://") {
```

Comment:
Have we settled on using `https://` protocol for grpc-tls, I thought we are aiming for `grpc+tls://`

Reply:
As discussed, leave as it is for now.